java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Request/ResolveRequest.java

```diff
@@ -14,6 +14,7 @@
 @SuperBuilder(builderMethodName = "resolveRequestBuilder", toBuilder = true)
 public class ResolveRequest extends MultifieldRequest {
   @Singular private List<Tag> tags;
+  private boolean generatePid;
```

Comment:
In the data-platform-core, the `generatedPid` was added to MultiFieldReq, and also this is used for both map and resolve. Move this field to MultifieldRequestt.

Reply:
its added here to make it available only for Resolve, and not for Enrich. If its added in MultifieldReq, it will be available for person.enrich also.